docs/release-notes/release-notes-0.15.1.md

```diff
@@ -115,6 +115,9 @@
 * [Re-initialise registered middleware index lookup map after removal of a 
   registered middleware](https://github.com/lightningnetwork/lnd/pull/6739)
 
+* [Bitcoind cookie file path can be specified with zmq
+  options](https://github.com/lightningnetwork/lnd/pull/6735)
```

Comment:
PR number is wrong in link!

Reply:
ah sorry, another point on the checklist